### PR TITLE
codec(ticdc): simple protocol DDL message encode 2 table schema

### LINKED ISSUE #10293: DDL message carries 2 schemas

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #10293

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
`None`
```

### CODE REVIEW

**Review by asddongmen — APPROVED** [review]

File: pkg/sink/codec/simple/message.go [hunk 1]
```diff
@@ -369,29 +393,28 @@ func newResolvedMessage(ts uint64) *message {
...
+		preSchema *TableSchema
 	)
 	// the tableInfo maybe nil if the DDL is `drop database`
 	if ddl.TableInfo != nil && ddl.TableInfo.TableInfo != nil {
 		schema = newTableSchema(ddl.TableInfo)
-		database = ddl.TableInfo.TableName.Schema
-		table = ddl.TableInfo.TableName.Table
-		schemaVersion = ddl.TableInfo.UpdateTS
+	}
+	if !ddl.IsBootstrap {
```
Comment (CharlesCheung96) [inline pkg/sink/codec/simple/message.go:403]: IsBootstrap should be removed.

Comment (3AceShowHand) [reply pkg/sink/codec/simple/message.go:403]: It will be removed in the next PR which is to support sending bootstrap at the processor side.

**Review by CharlesCheung96 — APPROVED** [review]

**Review by sdojjy — APPROVED** [review]

**Pull request merged**